.github/workflows/helm-multi-release.yaml

```diff
@@ -0,0 +1,183 @@
+          done
+        shell: bash
+        working-directory: ${{ inputs.charts-dir }}
+
+      - name: Commit and push updated charts
+        uses: bakdata/ci-templates/actions/commit-and-push@v1.6.0
+        with:
+          add-untracked: true
+          ref: ${{ env.ref }}
+          commit-message: "Update Chart versions"
+          github-username: ${{ secrets.github-username }}
+          github-email: ${{ secrets.github-email }}
+          github-token: ${{ secrets.github-token }}
+
+  package-new-artifacts:
+    strategy:
+      matrix:
+        subdir: ${{ fromJson(inputs.subdirs) }}
+    runs-on: ubuntu-22.04
+    needs: [update-version]
+    env:
+      charts-subdir: "${{ matrix.subdir }}"
+      artifact-dir: "${{ inputs.artifact-dir }}"
+      ref: ${{ github.event.repository.default_branch }}
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ env.ref }}
+
+      - name: Prepare artifacts
+        run: |
+          mkdir -p ${{ env.artifact-dir }}/${{ env.charts-subdir }}
+          helm package ./${{ inputs.charts-dir }}/${{ env.charts-subdir }} -d ${{ env.artifact-dir }}/${{ env.charts-subdir }}
+        shell: bash
+        working-directory: ${{ inputs.project-root-dir }}
+      
+      - name: Upload artifacts with action/upload
+        uses: actions/upload-artifact@v3.1.1
+        with:
+          name: ${{ env.charts-subdir }}
+          path: ${{ env.artifact-dir }}/${{ env.charts-subdir }}
+          retention-days: 1
+
+  publish-artifacts:
+    runs-on: ubuntu-22.04
+    needs: [package-new-artifacts]
+    env:
+      gh-pages: ${{ inputs.gh-pages-branch }}
+      root-dir: ${{ inputs.project-root-dir }}
+      artifact-dir: "${{ inputs.artifact-dir }}"
+      charts-dir: "${{ inputs.artifact-dir }}"
+
+    steps:
+      - name: Check out branch containing the artifatcs and the index.yaml file
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ env.gh-pages }}
+          
+      - name: Download new artifacts
+        uses: actions/download-artifact@v3.0.1
+        with:
+          path: ${{ env.artifact-dir }}
+      
+      - name: Merge contents if the artifacts already exist or Create new tree if not
+        run: |
+          for subdir in */; do
+            if [ -d "../$subdir" ]; then
+              echo "$subdir arleady exists Moving content of $subdir"
+              mv "$subdir"/* ../"$subdir"
+            else
+              echo "$subdir does not exist. Moving the whole directory $subdir"
+              mkdir ../"$subdir"
+              mv "$subdir"/* ../"$subdir"
+            fi
+          done
+          cd ..
+          rm -rf ${{ env.artifact-dir }}
+        shell: bash
+        working-directory: ${{ env.artifact-dir }}
+
+      - name: Generate index.yaml file
+        run: |
+          helm repo index .
+        shell: bash
+        working-directory: ${{ env.root-dir }}
```

Comment:
This always needs to be `.` and not the `project-root-dir` because we are on the gh-pages-branch right?

Reply:
thats a confusing naming by me. i mean chart dir and not root dir